src/brogue/IO.c

```diff
@@ -2686,10 +2688,14 @@ void executeKeystroke(signed long keystroke, boolean controlKey, boolean shiftKe
                 copyDisplayBuffer(dbuf, displayBuffer);
                 funkyFade(dbuf, &white, 0, 100, mapToWindowX(player.loc.x), mapToWindowY(player.loc.y), false);
             }*/
-            // DEBUG displayLoops();
-            // DEBUG displayChokeMap();
+            DEBUG displayLoops();
+            DEBUG displayChokeMap();
             DEBUG displayMachines();
-            //DEBUG displayWaypoints();
+            DEBUG displayWaypoints();
+            
+#ifdef LOG_LIGHTS
```

Comment:
The indentation looks a little off here

Reply:
Fixed the indentation on the `logLights()` line. Having no indentation for the preprocessor directives seems standard though.

Reply:
I presume @tmewett means he prefers the new indentation, rather than wanting no indentation?

Reply:
I mean that I agree it's typical for preprocessor directives to have no indentation - diff as-is looks good to me